build.psm1

```diff
@@ -848,6 +848,16 @@ function Start-PSPester {
         [switch]$IncludeFailingTest
     )
 
+    if (-not (Get-Module -ListAvailable -Name $Pester -ErrorAction SilentlyContinue)) 
```

Comment:
I actually prefer the `Test-Path` instead. With `-ListAvailable` the Pester version that comes installed with Windows 10 will be found. There are difference in those versions which might cause false errors in running tests.

Reply:
@adityapatwardhan Since the full path to the project's Pester module is being passed in here, I believe this would not be an issue:

<img width="1200" alt="screen shot 2017-10-10 at 2 22 11 pm" src="https://user-images.githubusercontent.com/1501176/31403205-74787dc4-adc6-11e7-9bdc-f6ad63e9c819.png">

However, if we only passed in `Pester`, I could see that being a cause for concern:

<img width="709" alt="screen shot 2017-10-10 at 2 24 13 pm" src="https://user-images.githubusercontent.com/1501176/31403293-b8a2656e-adc6-11e7-83a8-691675140f76.png">

Reply:
Makes sense. Misread it being Pester instead of $Pester.